upstairs/src/lib.rs

```diff
@@ -7015,12 +6961,6 @@ impl BlockIO for Guest {
     }
 
     async fn show_work(&self) -> Result<WQCounts, CrucibleError> {
```

Comment:
Could you leave behind a comment that this job will be sent even if the upstairs is not active?  Either here, or over in  `process_new_io()`?  There may come a time or reason to turn that off (or at least check)

Reply:
:+1:, done in 675f812